packages/clerk-js/src/core/devBrowserHandler.ts

```diff
@@ -157,13 +163,13 @@ export default function createDevBrowserHandler({
     if (devOrStgApi) {
       fapiClient.onBeforeRequest(request => {
         const devBrowserJWT = getDevBrowserJWT();
-        if (devBrowserJWT) {
-          request.url?.searchParams.set(DEV_BROWSER_SSO_JWT_PARAMETER, devBrowserJWT);
+        if (devBrowserJWT && request?.url) {
+          setDevBrowserJWTInURL(request.url, devBrowserJWT, true);
         }
       });
 
       fapiClient.onAfterResponse((_, response) => {
-        const newDevBrowserJWT = response?.headers?.get(DEV_BROWSER_SSO_JWT_HTTP_HEADER);
+        const newDevBrowserJWT = getDevBrowserJWTFromResponse(response);
         if (newDevBrowserJWT) {
           setDevBrowserJWT(newDevBrowserJWT);
```

Comment:
Same as above

Reply:
This is not the same method (the previous method was `setDevBrowserJWTInURL`)